### ISSUE TITLE
latest tag on docker hub doesn't point to latest

### ISSUE THREAD

**Issue by colemickens** [Azure/azure-cli-docker#29]
```
$ docker run -it microsoft/azure-cli:latest

root@0f0d3e07a721:/# azure --version
0.9.14 (node: 4.2.4)
```

expected:

```
$ docker run -it microsoft/azure-cli:latest

root@0f0d3e07a721:/# azure --version
0.9.19 (node: 4.2.4)
```

**Comment by colemickens** [comment]
Fix:

```
docker pull microsoft/azure-cli:0.9.19
docker tag microsoft/azure-cli:0.9.19 microsoft/azure-cli:latest
docker push -f microsoft/azure-cli:latest
```

**Comment by colemickens** [comment]
Except that `0.9.19` isn't published either.

**Comment by ahmetalpbalkan** [comment]
@colemickens nope it should be fixed on Docker Hub management UI. We don't `docker push` to automated build repos.

**Comment by colemickens** [comment]
Hm, okay. Is this being built automatically though? Why isn't 0.9.19 auto built/tagged then? Or does each new version need to be configured in the Hub Management UI?

**Comment by ahmetalpbalkan** [comment]
That's right, it's a manual process. There's not much automation Docker Hub is offering here.

**Comment by amarzavery** [comment]
This should be fixed per responses in #30.

**Comment by colemickens** [comment]
Thanks again @amarzavery.
